src/Symfony/Bridge/Twig/Tests/Command/LintCommandTest.php

```diff
@@ -134,7 +136,35 @@ public function testLintAutodetectsGithubActionEnvironment()
         }
     }
 
+    /**
+     * @dataProvider provideCompletionSuggestions
+     */
+    public function testComplete(array $input, array $expectedSuggestions)
+    {
+        $tester = new CommandCompletionTester($this->createCommand());
+
+        $this->assertSame($expectedSuggestions, $tester->complete($input));
+    }
+
+    public function provideCompletionSuggestions()
+    {
+        yield 'option' => [
+            ['--format', ''],
+            ['txt', 'xml', 'json', 'md'],
+        ];
```

Comment:
Here is the suggestions that is expected.
```suggestion
        yield 'option' => [['--format', ''], ['txt', 'json', 'github']];
```